src/main/java/com/velox/sloan/cmo/workflows/kapalibrary/ManualIndexAssignmentHandler.java

```diff
@@ -58,7 +59,11 @@ public PluginResult run() throws ServerException {
             }
             Integer plateSize = getPlateSize(attachedSamplesList);
             String sampleType = attachedSamplesList.get(0).getStringVal("ExemplarSampleType", user);
-            Double minAdapterVolInPlate = autohelper.getMinAdapterVolumeRequired(plateSize);
+            String recipe = attachedSamplesList.get(0).getStringVal("Recipe", user);
```

Comment:
I think this would cause a conflict when the samples later move through the Kapa Library Prep Workflow - manual index assignment step but off the top of my head I'm not sure what it should be changed to. I'll think about it!

Reply:
Let's discuss it on Tuesday.